srv/components/defaults.yaml

```diff
@@ -54,25 +46,7 @@ tmp_dir: /tmp
 
   # LEGACY REPOS STRUCTURE
 
-  {% set os_base_repo = salt["environ.get"]('CORTX_PLATFORM_BASE', "http://ssc-satellite1.colo.seagate.com/pulp/repos/EOS/Library/custom/CentOS-7/CentOS-7-OS/") %}
-    # url: http://mirrors.usc.edu/pub/linux/distributions/centos/7.7.1908/os/x86_64/
-  {% set os_extras_repo = salt["environ.get"]('CORTX_PLATFORM_EXTRAS', "http://ssc-satellite1.colo.seagate.com/pulp/repos/EOS/Library/custom/CentOS-7/CentOS-7-Extras/") %}
-    # url: http://mirrors.usc.edu/pub/linux/distributions/centos/7.7.1908/extras/x86_64/
-  {% set os_updates_repo = salt["environ.get"]('CORTX_PLATFORM_UPDATES', "http://ssc-satellite1.colo.seagate.com/pulp/repos/EOS/Library/custom/CentOS-7/CentOS-7-Updates/") %}
-    # url: http://mirrors.usc.edu/pub/linux/distributions/centos/7.7.1908/updates/x86_64/
-
-  {% set os_repos = [
-        {"id": "cortx_platform_base", "url": os_base_repo},
-        {"id": "cortx_platform_extras", "url": os_extras_repo},
-        {"id": "cortx_platform_updates", "url": os_updates_repo}
-      ]
-  %}
-
-  {% if "RedHat" in grains['os'] %}
-    {% set commons_repo = pillar['commons']['cortx_commons']['RedHat'] %}
-  {% else %}
-    {% set commons_repo = pillar['commons']['cortx_commons']['CentOS'] %}
-  {% endif %}
+ {% set common_repo = '/'.join([target_build, '3rd_party']) %}
```

Comment:
That should never happen since {target_build}/3rd_party means that it is a bundle distribution. And epel and saltstack are part of it.
If we want to drop the support for satellite urls then better just to remove that `# LEGACY REPOS STRUCTURE` block

Reply:
@andkononykhin2 
This needs rework agreed, but for now we need to make minimum changes to code
Let's discuss.